client/src/pages/Login.js

```diff
@@ -1,16 +1,21 @@
-import React from "react";
+import React, { useState } from "react";
 import styled from "styled-components";
 import { Button, Flex, WhiteSpace } from "antd-mobile";
 import { Link } from "react-router-dom";
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
I think we would better avoid this
because currently we are using Ant design + Styled components
The aim for the web app is mostly to focus on mobile, so I think it would be better not to use all these different libraries (currently we have Ant Deisgn, Ant Mobile, Material UI ... and probably still Grommet in there), plus our own base components. We have to take one approach and stay consistent.